crates/compiler/mono/src/layout/intern.rs

```diff
@@ -329,6 +329,10 @@ pub trait LayoutInterner<'a>: Sized {
         let doc = self.to_doc_top(layout, &alloc);
         doc.1.pretty(80).to_string()
     }
+
+    fn dbg_big<'r>(&'r self, layout: InLayout<'a>) -> dbg::Dbg<'a, 'r, Self> {
+        dbg::Dbg(self, layout)
+    }
```

Comment:
This looks cool! Thanks for doing it!

Would you mind adding a doc comment with things like
- an example of how people could use it (perhaps in the middle of some back end code somewhere)
- a mention of why it is not just a case of putting a `Debug` instance on `Layout`
- what the "big" is referring to

Reply:
addressed in [0af5929](https://github.com/roc-lang/roc/pull/4987/commits/0af592941116054eaed6414adbbd6e4319031b1a). renamed to dbg_deep